pkgs/by-name/tp/tplay/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  pkg-config,
+  openssl,
+  opencv,
+  clang,
+  libclang,
+  ffmpeg,
+  alsa-lib,
+  fetchFromGitHub,
+  rustPlatform
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "tplay";
+  version = "0.45";
+
+  src = fetchFromGitHub {
+    owner = "maxcurzi";
+    repo = pname;
+    rev =  "v${version}";
+    hash = "sha256-qt5I5rel88NWJZ6dYLCp063PfVmGTzkUUKgF3JkhLQk=";
+  };
+
+  cargoHash = "sha256-jgUKGV2Yg8+iF2wQZd1Z+QFfyJmywVTFVECUs+TK8zA=";
+  cargoPatches = [ ./cargo.diff ];
+  doCheck = false;
+
+  buildInputs = [
+    openssl.dev
+    alsa-lib.dev
+    libclang.lib
+    ffmpeg.dev
+    opencv
+  ];
+
+  nativeBuildInputs = [
+    pkg-config
+    clang
+  ];
+
+  env.LIBCLANG_PATH = "${libclang.lib}/lib";
+
+  meta = {
+    description = "Terminal Media Player";
+    homepage = "https://github.com/maxcurzi/tplay";
+    platforms = lib.platforms.linux;
+    license = with lib.licenses; [ mit ];
+    maintainers = with lib.maintainers; [  ];
```

Comment:
```suggestion
    maintainers = with lib.maintainers; [  ];
    mainProgram = "tplay";
```

Reply:
That not true. That behaviour is deprecated. See the comment on line 149: [https://github.com/NixOS/nixpkgs/blob/master/lib/meta.nix](https://github.com/NixOS/nixpkgs/blob/master/lib/meta.nix)